src/protocols/ios/ios.ts

```diff
@@ -592,10 +592,11 @@ export abstract class IOSProtocol extends ProtocolAdapter {
         let type;
         if (message.type === 'log') {
             switch (message.level) {
-                case 'log': type = 'log'; break;
+                case 'log': type = 'info'; break;
```

Comment:
Is the intention to "mute" the console.log messages or what's the intent of mapping log to info?

Reply:
Apparently the devtools won't show log type messages only info, warning and error.
I debugged devtools to understand, the messages are received, stored in the console array but not displayed, when they are transformed into info they are displayed. 
I didn't find the reason why log type message are filtered, maybe the label is wrong.
I know it's a hack, but very useful.